### ISSUE TITLE
[Bug][Grid] Not allowed aria-rowindex attribute axe error with no records

### ISSUE THREAD

**Issue by iadnonov** [telerik/kendo-react#2294]
## I'm submitting a...

* Bug report 

## Current behavior
When there are no records in the Grid, the no records template triggers a not allowed aria-rowindex attribute axe error.

## Expected behavior
When there are no records in the Grid, there should be no axe aria-rowindex error.

## Minimal reproduction of the problem with instructions
  1. Open the following [example](https://stackblitz.com/edit/react-dp2wut-chacgz?file=app%2Fmain.jsx).
  2. Scan the page with axe DevTools.

Ticket ID: 1654328

**Comment by silviyaboteva** [comment]
This pull request is included in version 8.2.0.
